### DISCUSSION TITLE
wwwroot/blog-post

### DISCUSSION THREAD

**Discussion by giscus** [bloggrammer/DevBlog#7] [Announcements]
# wwwroot/blog-post

Page description

http://localhost/post

**Comment by bloggrammer** [comment]
Testing Giscus

**Reply by botmasta** [reply]
Yeah, works like a charm! :)

**Comment by botmasta** [comment]
This works

**Reply by trinityimma** [reply]
Charming indeed

**Comment by trinityimma** [comment]
This is nice bloggrammer!
Well done
